tests/test_loan_transitions.py

```diff
@@ -387,7 +378,7 @@ def test_checkout_item(loan_created, db, params,
 
 
 def test_checkout_item_unavailable_steps(loan_created, db, params,
-                                         users, app, mock_can_be_requested):
+                                         users, app):
```

Comment:
Please fix all the signatures of test functions to be alike. Personally I am a fan of the following layout, the right alignment is not consistent.

```
def test_checkout_on_unavailable_item(
    loan_created, db, params, mock_is_item_available, mock_can_be_requested
):
```

Reply:
done